easybuild/easyconfigs/m/mrsFAST/mrsFAST-2.6.0.4-intel-compiler.patch

```diff
@@ -0,0 +1,20 @@
+replace hardcoded gcc by $CC so it can be built with intel toolchains
+Pablo Escobar - sciCORE
+
+diff -ru mrsfast-2.6.0.4.orig/Makefile mrsfast-2.6.0.4/Makefile
+--- mrsfast-2.6.0.4.orig/Makefile	2013-07-24 00:53:41.000000000 +0200
++++ mrsfast-2.6.0.4/Makefile	2014-05-07 11:35:36.394689000 +0200
+@@ -1,3 +1,5 @@
++CC=icc
```

Comment:
please change this to pick up `$CC` from the environment, and fall back to `gcc`

```
CC=${CC:-gcc}
```